pkgs/default.nix

```diff
@@ -32,7 +32,14 @@
           // extra))
         .${wine};
     in {
-      inherit (inputs.umu.packages.${system}) umu;
```

Comment:
This is technically a breaking change. Should there be an `umu` -> `umu-launcher` alias for end-users?

Probably **not a big deal**, any users using `packages.<system>.umu` would get an error that the flake output doesn't exist.

_If_ you think it's worth it, you could provide an alias:

```nix
umu = lib.warn "nix-gaming: `packages.${system}.umu` has been renamed to `packages.${system}.umu-launcher`." config.packages.umu-launcher;
```

IDK how nix-gaming has handled such renames in the past?

Reply:
We have this helper function for deprecations https://github.com/fufexan/nix-gaming/blob/master/lib/default.nix#L28.

Reply:
So:
```nix
umu = mkDeprecated "warn" config.packages.umu-launcher {
  name = "`umu`";
  target = "package";
  date = "2025-02-04";
  instructions = "Renamed to `umu-launcher`."
};
```
?

(Feedback from an outside perspective: it may be nice to have both `mkDeprecated` and `mkRename`. I also wonder about moving the first two params into the de-structured-attrs param?)

Reply:
Sure, we can do that.

And yes, that looks fine.